components/atoms/Accordion2.js

```diff
@@ -0,0 +1,87 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion2(props) {
+  const [openFlag, setOpenFlag] = useState(false);
+
+  function openSummary() {
+    setOpenFlag(!openFlag);
+  }
+
+  const arrowColor = "#335075";
+
+  return (
+    <div className="flex mb-0 ">
+      <details
+        className={`accordion flex-1 flex-wrap p-4 border border-gray-300 rounded bg-white ((${props.mainClass})? ${props.mainClass} : '') `}
```

Comment:
```suggestion
        className={`accordion flex-1 flex-wrap pt-3 px-2 border border-gray-300 rounded bg-white ((${props.mainClass})? ${props.mainClass} : '') `}
```
Just a suggestion to make the accordions look closer to what is in the video Steph took of the MVP.